install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2718,6 +2718,9 @@ clustermesh:
       # -- Security context to be added to clustermesh-apiserver etcd containers
       securityContext: {}
 
+      # -- lifecycle setting for the etcd container.
```

No periods for sentence fragments
```suggestion
      # -- lifecycle setting for the etcd container
```